.gitignore

```diff
@@ -14,3 +14,4 @@ spec/reports
 test/tmp
 test/version_tmp
 tmp
+Gemfile.lock
```

Comment:
I removed Gemfile.lock because it is not supossed to be in version control for gems.